dbms/src/Storages/Page/V3/BlobStore.h

```diff
@@ -74,9 +74,9 @@ class BlobStore : private Allocator<false>
             // After GC remove, empty files will be removed.
             READ_ONLY = 2,
 
-            // Big Blob file
-            // Only used to page size > config.file_limit_size
-            BIG_BLOB = 3
+            // Blob file size which is larger than config.file_limit_size.
+            // Won't be reused for other write.
+            ONE_SHOT = 3
```

Comment:
I wonder why we need a new type, `ONE_SHOT`. I think the common logic should handle the case that the real data size of a blob file is larger than size limit.

Reply:
Remove ONE_SHOT now.